packages/cadl-typescript/src/modelUtils.ts

```diff
@@ -334,10 +335,13 @@ function getSchemaForModel(
   usage?: SchemaContext[],
   needRef?: boolean
 ) {
-  const friendlyName = getFriendlyName(program, model);
+  const overridedModelName =
+    getProjectedName(program, model, "javascript") ??
```

Comment:
Why do we need  `getProjectedName(program, model, "javascript")`? We shouldn't allow renames on the REST API names.

Reply:
I think it would be safer if we removed both `getFriendlyName(program, model);` and `getFriendlyName(program, model);`

Reply:
Sure we could remove the logic for getFriendlyName. 

I added this because the model name will not be used for any serialization and deserialization. So I think it would be better to name the model with language specific name or friendly name than json one.